lms/djangoapps/instructor/views/tools.py

```diff
@@ -292,8 +294,9 @@ def generate_course_forums_d3(url_handle):
             previous[1] = number
         elif comment_type == 'Comment':
             previous[2] = number
+
     # return nothing if there's no data
-    if len(rows) == 1:
+    if not previous:
```

Comment:
I see, the logic appears to be the same, but I am okay with keeping it as in the PR